libs/src/services/discoveryProvider/DiscoveryProviderSelection.test.js

```diff
@@ -3,9 +3,10 @@ const assert = require('assert')
 const semver = require('semver')
 const DiscoveryProviderSelection = require('./DiscoveryProviderSelection')
 const DiscoveryProvider = require('.')
-const helpers = require('../../../tests/helpers')
+// const helpers = require('../../../tests/helpers')
```

Comment:
this is actually intentional for now, should prob add a comment why i did that. so basically importing helpers was causing an issue since it depends on generated files, which im not sure of the best way to handle. since this test wasn't even being run (and still isnt) for a while now, this is why i just patched it so i could include the other files that were missed in the test suite (the explicit '' in the package.json file brings a lot of them back)